mapping-variant-calling.md

```diff
@@ -40,43 +40,82 @@ Goal: execute a basic mapping
 
 ### Download and gunzip the reference:
 
+Here we are using open coding regions to do variant calling because we are working with mRNA sequences.
+It's important to think about what reference is appropriate for your experiment. Many biologically important
+variants exist in non-coding regions, so if we were looking at genomic sequences, it would be important to
+use a different reference such as the whole genome.
+
 ```
 curl -O https://downloads.yeastgenome.org/sequence/S288C_reference/orf_dna/orf_coding.fasta.gz
 gunzip orf_coding.fasta.gz
 ```
 
-and look at it:
+Let's take a look at our reference:
 
 ```
 head orf_coding.fasta
 ```
         
-### Prepare it for mapping:
+### Indexing: Prepare reference for mapping:
+
+Our first step is to index the reference genome for use by BWA. Indexing allows the aligner to quickly find potential
+alignment sites for query sequences in a genome, which saves time during alignment. Indexing the reference only has to 
+be run once. The only reason you would want to create a new index is if you are working with a different reference genome
+or you are using a different tool for alignment.
 
 ```
 bwa index orf_coding.fasta
 ```
         
 ### Map!
 
+We use an algorithm called `bwa mem` to perform mapping.
+
 ```
 bwa mem -t 4 orf_coding.fasta ERR458493.qc.fq.gz  > ERR458493.sam
 ```
         
+**What is the difference between Salmon and bwa mem?** In an earlier tutorial, we used Salmon "quasi-mapping" to generate
```

Comment:
@AstrobioMike please cross check and change as you see fit :)

Reply:
perfect!